zipkin-autoconfigure/storage-elasticsearch-http/src/main/java/zipkin/autoconfigure/storage/elasticsearch/http/ZipkinElasticsearchHttpStorageAutoConfiguration.java

```diff
@@ -45,8 +45,11 @@ StorageComponent storage(ElasticsearchHttpStorage.Builder esHttpBuilder) {
   ElasticsearchHttpStorage.Builder esHttpBuilder(
       ZipkinElasticsearchHttpStorageProperties elasticsearch,
       @Qualifier("zipkinElasticsearchHttp") OkHttpClient client,
-      @Value("${zipkin.storage.strict-trace-id:true}") boolean strictTraceId) {
-    return elasticsearch.toBuilder(client).strictTraceId(strictTraceId);
+      @Value("${zipkin.storage.strict-trace-id:true}") boolean strictTraceId,
+      @Value("${zipkin.query.lookback:86400000}") int namesLookback) {
```

Comment:
ps I'm ok to add this as a elasticsearch property instead. ex zipkin.storage.elasticsearch.names-lookback (we have similar things in cassandra)

Reply:
Add this as a elasticsearch property, may be a better idea. `zipkin.query.lookback` is a 32bit int variable which has a maximum of 2_147_483_647(ms) that is about 59.6621 hours. It is really not enough. 
A zipkin.storage.elasticsearch.names-lookback property in secs or even days can be great. Or may be you can use a 64bit variable for `zipkin.query.lookback`.
ps. Should this be a new issue?